Makefile

```diff
@@ -36,19 +39,19 @@ services:
 	docker-compose up -d
 
 .PHONY: dev
-dev: build/manifest.json
+dev: build/manifest.json ${TOX}
```

Comment:
`%{TOX}` makes `~/.pyenv/versions/3.6.8/bin/tox` a prerequisite of `dev`. And `~/.pyenv/versions/3.6.8` (`${PYTHON}`) is a prerequisite of `${TOX}`. So this will trigger the `${PYTHON}` (install Python) and then the `${TOX}` (install tox) recipes whenever `make dev` is run.